src/lib/y2partitioner/widgets/pervasive_key.rb

```diff
@@ -0,0 +1,73 @@
+# Copyright (c) [2025] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "cwm"
+
+module Y2Partitioner
+  module Widgets
+    # Widget to display the full verification pattern for the selected master key, in case the
+    # pattern is too long to be displayed at the corresponding selector
+    class PervasiveKey < CWM::ReplacePoint
+      # Internal widget showing the full verification pattern split into several lines
+      class Label < CWM::CustomWidget
+        # Constructor
+        def initialize(widget_id, key)
+          super()
+          self.widget_id = widget_id
+          @key = key
+        end
+
+        # @macro seeCustomWidget
+        def contents
+          lines = [@key[0..33], "  #{@key[34..-1]}"]
```

Comment:
will second part of key fit screen? I think we had in yast2.rpm something to split it into array

Reply:
Yes. The string of the key verification pattern can be either 18 characters ("0X" + 16 chars) or 66 characters ("0x" + 64 chars).

Reply:
It should be better explained now.